pom.xml

```diff
@@ -36,15 +36,68 @@
     </properties>
 
     <modules>
-        <module>cli</module>
         <module>convert</module>
         <module>confluence-client</module>
+        <module>core</module>
+        <module>cli</module>
     </modules>
 
     <build>
-        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
-        <testSourceDirectory>${project.basedir}/src/test/kotlin</testSourceDirectory>
         <plugins>
+            <plugin>
```

Comment:
is it some sort of preparation for publishing stuff to maven central? If so, let's also postpone this change to make PR better scoped to main purpose of simply splitting cli module into 2..